LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -231,21 +241,22 @@ public void SplitTenderExample_LevelUp_Then_Cash_With_Adjusted_Exemption_And_Adj
                 $"Expected: {expectedProposedOrderValues}" + Environment.NewLine +
                 $"Actual: {proposedOrderValues}");
 
-            // available discount amount $1
-            int availableDiscountAmount = 100;
+            // available discount amount $10.00
+            int availableDiscountAmount = 1000;
 
-            // POS applies $1.00 pretax discount to check and updates subtotal and tax
+            // POS applies $10.00 pretax discount to check and updates subtotal and tax
             check.ApplyDiscount(availableDiscountAmount);
 
-            // Create completed order: expected values
-            // spend amount unchanged
-            // tax amount unchanged
-            // exemption amount *changed*
+            // $ 0.00 (Subtotal Remaining)
+            // $ 1.00 (Tax)
+            // $ 1.00 (Total)
+            // $11.00 (Total + Discount Applied) 
```

Comment:
This is essentially $1 of linked payment from the user's account and $10 of discount credit?